tutorials/international/index.md

```diff
@@ -0,0 +1,281 @@
+class MyLocalizations {
+  MyLocalizations(this.locale);
+
+  final Locale locale;
+
+  static Future<MyLocalizations> load(Locale locale) {
+    return initializeMessages(locale.toString())
+      .then((Null value) {
+        return new MyLocalizations(locale);
+      });
+  }
+
+  static MyLocalizations of(BuildContext context) {
+    return Localizations.of<MyLocalizations>(context, MyLocalizations);
+  }
+
+  String get helloWorld => Intl.message('helloWorld', name: 'helloWorld', locale: locale.toString());
+  // ... more Intl.message() methods like get helloWorld
+}
+```
+
+A class based on the `intl` package imports a generated message catalog that provides the `initializeMessages()` function and the per-locale backing store for `Intl.message()`. The message catalog is produced by an `intl` tool that analyzes the source code for classes that contain `Intl.message()` calls. In this case that would just be the `MyLocalizations` class.
+
+One could choose another approach for loading localized resources and looking them up while still conforming to the structure of MyLocalizations. The An Alternative Class for the App's Localized Resources below provides such an example.
+
+An instance of `MyLocalizations` is created by a simple `LocalizationsDelegate`:
+
+```dart
+class MyLocalizationsDelegate extends LocalizationsDelegate<MyLocalizations> {
+  @override
+  Future<MyLocalizations> load(Locale locale) => MyLocalizations.load(locale);
+
+  @override
+  bool shouldReload(MyLocalizationsDelegate old) => false;
+}
+```
+
+An instance of `MyLocalizationsDelegate` is provided to the application's
+[`MaterialApp`](https://docs.flutter.io/flutter/material/MaterialApp-class.html)
+(or its [`WidgetsApp`](https://docs.flutter.io/flutter/widgets/WidgetsApp-class.html)
+for non-material applications):
+
+```dart
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return new MaterialApp(
+      localizationsDelegates: <LocalizationsDelegate>[
+        new MyLocalizationsDelegate()
+      ],
+      supportedLocales: <Locale>[
+        const Locale('en', 'US'), // US English
+        const Locale('fr', 'CA'), // Candian French
+      ],
+      child: home: new Scaffold(
+        appBar: new AppBar(
+          title: new Text(MyLocalizations.of(context).helloWorld),
+          ...
+        ),
+      ),
+    );
+  }
+}
+```
+
+As you can see, the app's `AppBar` title comes from an instance of
+`MyLocalizations` produced by `MyLocalizations.load()` via
+`MyLocalizationsDelegate`. If the device's locale changes, then the
+app will be rebuilt with a new `MyLocalizations` object, one
+constructed for the new locale. Any reference to
+`MyLocalizations.of()` anywhere in the app will causes its context to
+be rebuilt if the locale changes.
+
+### The App's `supportedLocales` Parameter
+
+Although the Flutter's Material library includes support for about 16
+languages, only English language translations are available by
```

Comment:
as a reader, I'm left asking: "how does Flutter support more languages, if only english is available" ?